go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -367,7 +367,8 @@ func (tsv *TabletServer) initACL(tableACLConfigFile string, enforceTableACLConfi
 			tsv.ClearQueryPlanCache()
 		},
 	)
-	if err != nil {
+	// Log failure if either there was a problem loading the ACL, or if the ACL is empty
+	if err != nil || tableacl.GetCurrentConfig().String() == "" {
```

Comment:
I'm not sure if this is the correct place to perform this validation. I think a config file containing an empty JSON object should be treated as "valid", but a truncated file seems like a pretty clear configuration mistake. 🤔

Reply:
Addressed in [eca5d61](https://github.com/vitessio/vitess/pull/17274/commits/eca5d61c0e2157c18d969df6799be1d00396451c)